File: Examples/CSharp/ApplyLicense/ApplyLicenseUsingFileStream.cs
using Aspose.CAD;
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;

namespace Aspose.CAD.Examples.CSharp.ApplyLicense
{
    public class ApplyLicenseUsingFileStream
    {
        public static void Run()
        {
            // ExStart:ApplyLicenseUsingFileStream
            // Set path of the license file, i.e. c:\temp\
            string dataDir = @"c:\temp\";
            // Load an existing file in the stream
            FileStream LicStream = new FileStream(dataDir + "Aspose.CAD.lic", FileMode.Open);

            License license = new License();
            license.SetLicense(LicStream);
            // ExEnd:ApplyLicenseUsingFileStream
        }

    }
}

File: Examples/VisualBasic/ApplyLicense/ApplyLicenseUsingFileStream.vb
Imports Aspose.CAD
Imports System.Collections.Generic
Imports System.IO
Imports System.Linq
Imports System.Text

Namespace ApplyLicense
    Public Class ApplyLicenseUsingFileStream
        Public Shared Sub Run()
            ' ExStart:ApplyLicenseUsingFileStream
            ' Set path of the license file, i.e. c:\temp\
            Dim dataDir As String = "c:\temp\"
            ' Load an existing file in the stream
            Dim LicStream As New FileStream(dataDir & Convert.ToString("Aspose.CAD.lic"), FileMode.Open)

            Dim license As New License()
            license.SetLicense(LicStream)
            ' ExEnd:ApplyLicenseUsingFileStream
        End Sub

    End Class
End Namespace
